### ISSUE TITLE
Improve accessibility for remaining tutorial pages

### ISSUE THREAD

**Issue by Kfoster14** [ausaccessfed/dev-portal#196]
Accessibility can be improved for the tutorial content:

- Background and foreground colors do not have a sufficient contrast ratio (https://dequeuniversity.com/rules/axe/4.10/color-contrast?application=axeAPI)
- Heading elements are not in a sequentially-descending order
- Link underlines should have bottom padding and be of an acceptable colour (https://www.w3.org/TR/2008/NOTE-WCAG20-TECHS-20081211/working-examples/G183/link-contrast.html)
- Code blocks do not have a sufficient contrast ratio.
- Duplicate ids found for tutorial series

Use https://github.com/pa11y/pa11y for testing
